src/components/SingleColCard/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { Row, Col, Image, Button } from "antd";
+import { NavLink } from "react-router-dom";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const SingleColCard = ({ post, order, isTesting }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Row className="justify-content-md-center articleContainer2">
```

Comment:
whats the difference between singleColCard and singleblogcard

Reply:
it's same, when I uploaded 3 more images to wordpress however it's not getting from it and my wordpress page freezing because of my computer. In addition, I open two component and one of is row other one if column.

Reply:
@gizemcdeniz if they are the same components then why do we have 2 copies of it why not just use one of it ?